src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -130,120 +104,135 @@ private static void addAdditionalItemProperties(JsonObject request, Item item) {
+        copyProperty(address, jsonObject, "city");
+        copyProperty(address, jsonObject, "region");
+        copyProperty(address, jsonObject, "postalCode");
+        copyProperty(address, jsonObject, "countryId");
 
-        requestWithAdditionalInformation.put("deliveryAddress", mappedAddress);
-      }
-    }
+        requestJO.put("deliveryAddress", jsonObject);
+      });
   }
 
   private static void removeDeliveryAddress(JsonObject requestRepresentation) {
     if (requestRepresentation == null) {
+      log.info(UNABLE_TO_REMOVE_DELIVERY_ADDRESS_MSG);
       return;
     }
     requestRepresentation.remove("deliveryAddress");
   }
 
   private static void addAdditionalLoanProperties(JsonObject request, Loan loan) {
-    if(loan == null || loan.isClosed()) {
-      String reason = null;
-      if(loan == null) { reason = "null"; } else { reason = "closed"; }
-      log.info("Unable to add loan properties to request {}, loan is {}",
-          request.getString("id"), reason);
+    if (loan == null || loan.isClosed()) {
+      String reason = isNull(loan) ? "null" : "closed";
+      log.info(UNABLE_TO_ADD_LOAN_PROPERTIES_MSG, request.getString("id"), reason);
       return;
     }
+
     JsonObject loanSummary = request.containsKey("loan")
       ? request.getJsonObject("loan")
       : new JsonObject();
-    
-    if(loan.getDueDate() != null) {
+
+    if (loan.getDueDate() != null) {
       String dueDate = loan.getDueDate().toString(ISODateTimeFormat.dateTime());
       loanSummary.put("dueDate", dueDate);
-      log.info("Adding loan properties to request {}", request.getString("id"));
+      log.info(ADDING_LOAN_PROPERTIES_MSG, request.getString("id"));
     }
-    
+
     request.put("loan", loanSummary);
   }
-  
+
   private static void addAdditionalServicePointProperties(JsonObject request, ServicePoint servicePoint) {
-    if(servicePoint == null) {
-      log.info("Unable to add servicepoint properties to request {},"
-          + " servicepoint is null", request.getString("id"));
+    if (servicePoint == null) {
+      log.info(UNABLE_TO_ADD_SERVICEPOINT_MSG, request.getString("id"));
       return;
     }
 
     JsonObject spSummary = request.containsKey("pickupServicePoint")
-        ? request.getJsonObject("pickupServicePoint")
-        : new JsonObject();
+      ? request.getJsonObject("pickupServicePoint")
+      : new JsonObject();
+
     spSummary.put("name", servicePoint.getName());
     spSummary.put("code", servicePoint.getCode());
     spSummary.put("discoveryDisplayName", servicePoint.getDiscoveryDisplayName());
     spSummary.put("description", servicePoint.getDescription());
     spSummary.put("shelvingLagTime", servicePoint.getShelvingLagTime());
-    spSummary.put("pickupLocation", servicePoint.getPickupLocation());    
-    
+    spSummary.put("pickupLocation", servicePoint.getPickupLocation());
+
     request.put("pickupServicePoint", spSummary);
   }
+
```

Comment:
They are just moved to the bottom. If you want to return them back I will do it

Reply:
Please move them back unless you have other reasons to move them? It's easier to see the differences when they are at their original position.

Reply:
This is only one reason. Public methods should be in one place ether at the top or at the bottom of a file. I prefer placing them at the bottom. It gives me additional opportunity to have a look at class structure and code.

Reply:
Good to know the rationale, thanks. Top or bottom doesn't matter to me, as long as we're consistent. It looks to me like most of the classes in the project have the private methods on the bottom.  @mreno-EBSCO or @marcjohnson-kint do you have any thoughts?

Reply:
I prefer public methods at the top. But I am not sure about consistency. I find it useful to have private methods that are only used by a few public methods near the methods that use them so you don't have to jump around in the file. With IDEs these days, that normally doesn't matter as much. Just old habits.

I'm not sure it is worth strictly enforcing ordering, but if we do, general consensus is public above package protected above protected above private. Same with variables, where static variables are above member variables. The argument here is that ordering by accessibility (most to least) allows for easier understanding of the code that others may interact with.

Reply:
The [FOLIO official](https://dev.folio.org/guidelines/contributing/#coding-style) stance is to follow the [Sun/Oracle guidelines](https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html#1852) state:

> These methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.

Hence it does not advocate grouping by scope.

Personally, I prefer public / package methods towards the top, as they are likely to be the starting point for a developer when reading the code.

And I've found that grouping by functionality can be brittle when a private method is reused amongst multiple methods. 

I think confusion about grouping is a weak signal that there are multiple responsibilities within a class, and this might start to hint at extracting a class in order to organise by functionality or behaviour.

I tend not to give much feedback on aspects like this, given that my personal beliefs conflict with the official guidance :-/

@skoczko Is this something that you think we need to offer expanded guidance on, outside of the Java/Sun documentation that we reference?

Reply:
Just in case, I returned public methods last week. Now they are at the top.
I think this project is required more stronger code formatter and more modern then Sun/Oracle guidelines  "The last revision to this document was made on April 20, 1999" 
It makes me smile 😄

Reply:
@Anatolii-Starkov Please, feel free to raise this feedback as a JIRA issue for further discussion.

I'm only trying to share the context of what the project had already decided.